tests/bluetooth/tester/btp_spec.txt

```diff
@@ -1111,6 +1111,74 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
+	Opcode 0x1c - Get Attribute
+
+		Controller Index:	<controller id>
+		Command parameters:	Start Handle (2 octets)
+					End Handle (2 octets)
+					Attribute Type (1 octet)
```

Comment:
Im not sure I understand what this type means, usually the GATT attribute type is defined by its UUID so this might be confusing for anyone familiar with how GATT discover works, for instance, looking up for a Primary service type would have the UUID as value but both Characteristic Value and Descriptor the UUID is their type. Note that over the air the GATT discover is able to replicate a database so we could for exampled add local browsing support for BlueZ as was done with sdptool, that way we can open a socket with bluetoothd and browse locally using the same commands that a peer would use over the air.

Reply:
This attribute type parameters are defined below starting from 1125. GATT defines 6 different attribute types that can be distinguished using Attribute Type parameter. I know that in GATT defined attribute declarations Attribute Type is UUID. Hovewer every attribute has it's own UUID so this is may be specified in UUID parameter (see 1121) and I don't mean the 'type UUID' here.
I can extend command description if you want.
You could get eg. all the primary services (type 0x01) with wanted UUID. If you don't provide UUID, all primary services will be returned.
I don't want to end up with 6 different commands.

Reply:
@Vudentz I changed my mind, and I can say that You have absolute right about this Type parameter. Doing this GATT like will simplify BTP command and it's implementation on tester application. I will push regarding changes soon.